src/user-management/user-management.spec.ts

```diff
@@ -426,6 +450,53 @@ describe('UserManagement', () => {
     });
   });
 
+  describe('getPasswordReset', () => {
+    it('sends a Get PaswordReset request', async () => {
+      fetchOnce(passwordResetFixture);
+      const passwordReset = await workos.userManagement.getPasswordReset(
+        passwordResetId,
+      );
+      expect(fetchURL()).toContain(
+        `/user_management/password_reset/${passwordResetId}`,
+      );
+      expect(passwordReset).toMatchObject({
+        id: 'password_reset_01H5JQDV7R7ATEYZDEG0W5PRYS',
+        userId: 'user_01H5JQDV7R7ATEYZDEG0W5PRYS',
+        email: 'dane@workos.com',
+        passwordResetToken: 'Z1uX3RbwcIl5fIGJJJCXXisdI',
+        passwordResetUrl:
+          'https://your-app.com.com/reset-password?invitation_token=Z1uX3RbwcIl5fIGJJJCXXisdI',
+        expiresAt: '2023-07-18T02:07:19.911Z',
+        createdAt: '2023-07-18T02:07:19.911Z',
+      });
+    });
+  });
+
+  describe('createMagicAuth', () => {
+    it('sends a Create Magic Auth request', async () => {
+      fetchOnce(passwordResetFixture);
+
+      const response = await workos.userManagement.createPasswordReset({
+        email: 'dane@workos.com',
+      });
+
+      expect(fetchURL()).toContain('/user_management/password_reset');
+      expect(fetchBody()).toEqual({
+        email: 'dane@workos.com',
+      });
+      expect(response).toMatchObject({
+        id: 'password_reset_01H5JQDV7R7ATEYZDEG0W5PRYS',
+        userId: 'user_01H5JQDV7R7ATEYZDEG0W5PRYS',
+        email: 'dane@workos.com',
+        passwordResetToken: 'Z1uX3RbwcIl5fIGJJJCXXisdI',
+        passwordResetUrl:
+          'https://your-app.com.com/reset-password?invitation_token=Z1uX3RbwcIl5fIGJJJCXXisdI',
```

Comment:
Ditto, if above valid:
```suggestion
          'https://your-app.com/reset-password?token=Z1uX3RbwcIl5fIGJJJCXXisdI',
```